File: programovani/ulohy/files/modelovani-obrazcu-ii/trojuhelnik-90.js
// Deklarace a inicializace
const vel = 6; // Inicializace velikosti trojuhelniku

// Vnejsi cyklus pro radky
for(var i = vel; i >= 1; i--) {
    
    // Inicalizujeme promennou pro vypsani radky
    var rad = "";

    // Vnitrni cyklus pro jednotlive hvezdicky
    for(var j = i; j >= 1; j--)

        // Pridame do textoveho retezce hvezdicku
        rad += "* ";

    // Vypiseme radku
    console.log(rad);

}

File: programovani/ulohy/files/modelovani-obrazcu-ii/trojuhelnik-90.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace
    const int vel = 6; // Inicializace velikosti trojuhelniku

    // Vnejsi cyklus pro radky
    for(int i = vel; i >= 1; i--) {
        
        // Vnitrni cyklus pro jednotlive hvezdicky
        for(int j = i; j >= 1; j--)

            // Vypiseme hvezdicky
            cout << "* ";

        // Vypiseme novou radku
        cout << endl;

    }

}
